### PR TITLE
added-comments-and-like-functionality

### PR DESCRIPTION
**Pull request by hrishi82**
Added
- Add and delete posts
- Add and delete comments from posts
- Like and dislike post

### CODE REVIEW

File: src/index.css [hunk 1]
```diff
@@ -173,8 +174,11 @@ body{
     border: transparent;
   }
 
-
-
+  .liked{
+    color: red;
+    cursor: pointer;
```
Comment (Parul1999) [inline src/index.css:179]: Instead of red can use css variables.

File: src/pages/homepage/PostCard/PostCard.jsx [hunk 2]
```diff
@@ -1,53 +1,216 @@
...
+        if (delresp.status === 200 || delresp.status === 201) {
+          dispatch({ type: "SET_ALL_POSTS", payload: delresp.data.posts });
   
+          toast.error("Post Deleted!", {
+            position: "bottom-right",
+            autoClose: 1500,
+          });
+        }
+      } catch (err) {
+        console.log(err);
```
Comment (Parul1999) [inline src/pages/homepage/PostCard/PostCard.jsx:76]: Suggestion : we can show failed error in the ui.

File: src/pages/homepage/PostCard/postcard.css [hunk 3]
```diff
@@ -57,21 +61,77 @@
...
+/************** Card Option Modal ************/
+
+.post-options-modal-container{
+    position:absolute;
+    border-radius: 0.3rem;
+    top:0.5rem;
+    right: 1rem;
+    overflow: hidden;
+    /* padding: 0.5rem; */
+    border: 1px solid black;
```
Comment (ramalingaa) [inline src/pages/homepage/PostCard/postcard.css:123]: try using color varibales

**Review by ramalingaa — APPROVED** [review]
Code looks good. PR is good to merge

File: src/index.css (hunk 1 above)
Comment (hrishi82) [reply src/index.css:179]: sure

File: src/pages/homepage/PostCard/PostCard.jsx (hunk 2 above)
Comment (hrishi82) [reply src/pages/homepage/PostCard/PostCard.jsx:76]: I see, didn't think about it that way. Will look into it!

File: src/pages/homepage/PostCard/postcard.css (hunk 3 above)
Comment (hrishi82) [reply src/pages/homepage/PostCard/postcard.css:123]: Sure

**Pull request merged**